pontoon/base/views.py

```diff
@@ -316,16 +322,13 @@ def get_translations_from_other_locales(request):
     ).exclude(locale=locale)
 
     preferred = translations.filter(locale__in=preferred_locales)
```

Comment:
If the user isn't authenticated, this will break, because `preferred_locales` isn't defined. You'll need to execute all the code related to preferred locales only for authenticated users.